pandas/_libs/window.pyx

```diff
@@ -1242,43 +982,56 @@ def roll_min(ndarray[numeric] values, int64_t win, int64_t minp,
           is below this, output a NaN
     index: ndarray, optional
        index for window computation
+    closed: 'right', 'left', 'both', 'neither'
+            make the interval closed on the right, left,
+            both or neither endpoints
     """
-    return _roll_min_max(values, win, minp, index, is_max=0, closed=closed)
+    return _roll_min_max_variable(values, start, end, minp, is_max=1)
 
 
-cdef _roll_min_max(ndarray[numeric] values, int64_t win, int64_t minp,
-                   object index, object closed, bint is_max):
+def roll_min_fixed(ndarray[float64_t] values, ndarray[int64_t] start,
+                   ndarray[int64_t] end, int64_t minp, int64_t win):
     """
-    Moving min/max of 1d array of any numeric type along axis=0
-    ignoring NaNs.
+    Moving max of 1d array of any numeric type along axis=0 ignoring NaNs.
+
+    Parameters
+    ----------
+    values: numpy array
```

Comment:
space after colon, in these files we usually specify the dtype as if it were a cython annotation, so `np.ndarray[np.float64]`